using-mobile-extensions/adobe-target-vec.md

```diff
@@ -0,0 +1,144 @@
+# Adobe Target - VEC
+This extension enables Target VEC (Visual Experience Composer) functionality. The Target VEC for Native Mobile Apps lets you create activities and personalize content on native mobile apps in a do-it-yourself fashion without continuous development dependencies and app-release cycles.
+
+{% hint style="info" %}
+Review the following Target VEC functionality documentation for context before implementation at these links:
+
+* [Mobile App Visual Experience Composer](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/composer/mobile-visual-experience-composer.html)
+* [Setup Mobile VEC for Android](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/composer/mobile-visual-experience-composer-android.html)
+* [Setup Mobile VEC for iOS](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/composer/mobile-visual-experience-composer-ios.html)
+* [Set up click tracking in the Mobile VEC](https://docs.adobe.com/content/help/en/target/using/implement-target/mobile-apps/composer/set-up-click-tracking-in-the-mobile-vec.html)
+{% endhint %}
+
+To get started with Target VEC, follow these steps:
+
+1. Configure the Adobe Target - VEC extension. The VEC extension is dependent on the Adobe Target Extension. Make sure Adobe Target extension is already configured and enabled.
```

Comment:
Please change to:

To get started, complete the following steps:

Configure the Target VEC extension.

The The VEC extension is dependent on the Adobe Target Extension. Make sure Adobe Target extension is already configured and enabled.

Add the Target VEC Extension to your app. --> missing a period at the end.

Select one of the following implementation methods:

Auto-Fetch Target Activities, where the options are:

To fetch in a blocking call (background is OFF)
To fetch in an asynchronous mode (background is ON)
Fetch Target activities programmatically

Handle Target workspace restrictions